.github/workflows/ci-ai.yml

```diff
@@ -60,7 +60,7 @@ jobs:
               run: UV_PROJECT_ENVIRONMENT=$pythonLocation uv sync --frozen --dev
 
             - name: Install Rust
-              uses: dtolnay/rust-toolchain@6691ebadcb18182cc1391d07c9f295f657c593cd # 1.88
+              uses: dtolnay/rust-toolchain@0b1efabc08b657293548b77fb76cc02d26091c7e
```

Comment:
Can we add the action version back here to make this easier to bump in the future?

Reply:
I've removed it because this is not the action version. 

The action author was apparently very clever and made the tags based on the toolchain. So when you wanna pick 1.88 you use `dtolnay/rust-toolchain@1.88`. This doesn't mean you get `1.88` of the action but of the toolchain 🫠 

I should probably add comments to note that.